packages/data-stores/src/domain-suggestions/queries.ts

```diff
@@ -70,7 +70,7 @@ export function useGetWordPressSubdomain( paidDomainName: string ) {
 		quantity: 1,
 		include_wordpressdotcom: true,
 		include_dotblogsubdomain: false,
-		only_wordpressdotcom: true,
+		only_wordpressdotcom: false,
```

Comment:
This mimicks the current approach used to generate a domain in the domains step. Since we negate the dotblog parameter with a flag, this will conitnue to provide 1 wpcom subdomain